ui/src/app/views/account-creation/component.html

```diff
@@ -0,0 +1,55 @@
+<div style="padding: 1rem;">
+  <div *ngIf="!accountCreated" class="form-area">
+    <div class="form-section">
+      <label>Given Name:</label>
+      <input type="text" class="input" autofocus [(ngModel)]="givenName">
+    </div>
+    <div class="form-section">
+      <label>Family Name:</label>
+      <input type="text" class="input" [(ngModel)]="familyName">
+    </div>
+    <div class="form-section">
+      <label>Username:</label>
+      <input type="text" class="input" [(ngModel)]="username">
+    </div>
+    <div class="form-section">
+      <label>Contact Email:</label>
+      <input type="text" class="input" [(ngModel)]="contactEmail">
+    </div>
+    <div class="form-section">
+      <label>Password:</label>
+      <input type="password" class="input" [(ngModel)]="password">
+    </div>
+    <div class="form-section">
+      <label>Password Again:</label>
+      <input type="password" class="input" [(ngModel)]="passwordAgain">
+    </div>
+    <div class="form-section">
+      <label>Invitation Key:</label>
+      <input type="text" class="input" [(ngModel)]="invitationKey">
+    </div>
+    <div class="form-section">
+      <button (click)="createAccount()" class="btn btn-primary">Create Account</button>
+      <span class="spinner spinner-inline" *ngIf="showSpinner">
```

Comment:
For the `ngIf` can we call the boolean something like `creatingAccount`? (Just for more clarity on what's happening? Mark has been asking me to do this with my PRs and I think its a good idea in general.)

Reply:
Good call. This was a premature generalization.